examples/chat-with-history.py

```diff
@@ -0,0 +1,37 @@
+from ollama import chat
+
+
+messages = [
+  {
+    'role': 'user',
+    'content': 'Why is the sky blue?',
+  },
+  {
+    'role': 'assistant',
+    'content': "The sky is blue because of the way the Earth's atmosphere scatters sunlight.",
+  },
+  {
+    'role': 'user',
+    'content': 'What is the weather in Tokyo?',
+  },
+  {
+    'role': 'assistant',
+    'content': 'The weather in Tokyo is typically warm and humid during the summer months, with temperatures often exceeding 30°C (86°F). The city experiences a rainy season from June to September, with heavy rainfall and occasional typhoons. Winter is mild, with temperatures rarely dropping below freezing. The city is known for its high-tech and vibrant culture, with many popular tourist attractions such as the Tokyo Tower, Senso-ji Temple, and the bustling Shibuya district.',
+  },
+]
+
+while True:
+  user_input = input('Chat with history: ')
+  response = chat(
+    'llama3.2',
+    messages=messages
+    + [
+      {'role': 'user', 'content': user_input},
+    ],
+  )
```

Comment:
This doesn't add the user input as a message for future calls to chat

```suggestion
  user_input = input('Chat with history: ')
  messages.append({
    'role': 'user',
    'content': user_input,
  })
  response = chat(
    'llama3.2',
    messages=messages,
  )
```